tests/ui/pages/desktop/devhub.py

```diff
@@ -133,7 +133,7 @@ class Header(Region):
 
         _register_link_locator = (
             By.CSS_SELECTOR,
-            '.DevHub-Navigation-Register > a:nth-child(1)'
+            'li.show-on-desktop:nth-child(7) > a:nth-child(1)'
```

Comment:
Why is that necessary? The class `DevHub-Navigation-Register` still exists, these pages haven't changed since forever.

Reply:
I think this was a change due to the django toolbar. It should be able to be reverted.